ci_support/run_docker_build.sh

```diff
@@ -50,58 +50,58 @@ conda info
     export CONDA_PY=27
     set +x
     conda build /recipe_root --quiet || exit 1
-    
+
     /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
-    
```

Comment:
My bad. My text editor is configured to do that. (But we should fix that in the template, no?)